CHANGELOG.md

```diff
@@ -82,6 +82,9 @@
 - Segregated MMDS documentation in MMDS design documentation and MMDS user
   guide documentation.
 - The logger `level` field is now case-insensitive.
+- Updated firecracker --version flag to include `git describe --dirty` output,
```

Comment:
If we do it for `jailer` as well, update entry here too.

Reply:
I d rephrase this entry to something more concise and clear:

"`firecracker --version` gets updated on each `devtool build` to the output of `git describe --dirty`.
Also, if we keep the long version of the command (i.e porcelain stuff) this is not available anymore.

Reply:
edited